tests/std/tests/GH_004879_span_constructed_by_basic_ispanstream/test.compile.pass.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// GH-4879 <spanstream>: The span constructed by basic_ispanstream's range constructor may be ill-formed

```

Comment:
This seems pretty tiny to justify a separate test. Could we just add it to `tests\std\tests\P0448R4_spanstream\test.cpp`?